bigchaindb/__init__.py

```diff
@@ -45,7 +45,22 @@
         'private': None,
     },
     'keyring': [],
-    'backlog_reassign_delay': 120
+    'backlog_reassign_delay': 120,
+    'log': {
```

Comment:
If that is ok to leave it there for now, as it serves as documentation. Once we have documentation elsewhere we can remove it, and also, in the event that we would move the config file format to yaml, (see issue https://github.com/bigchaindb/bigchaindb/issues/1220) the comments could be carried over in the yaml file.